### PR TITLE
[14.0][IMP] account_payment_order: Improved a create_account_payment_line method.

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**
Improved a create_account_payment_line method.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You already have the move in `self`.

File: account_payment_order/models/account_move.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def create_account_payment_line(self):
                 raise UserError(_("No Payment Mode on invoice %s") % move.name)
             for payment_mode in payment_modes:
                 payorder = apoo.search(
-                    self.get_account_payment_domain(payment_mode), limit=1
+                    self.get_account_payment_domain(payment_mode, move), limit=1
```
Comment (pedrobaeza) [inline account_payment_order/models/account_move.py]:
```suggestion
                    move.get_account_payment_domain(payment_mode), limit=1
```

**Review by pedrobaeza — APPROVED** [review]

**Review by murtuzasaleh — APPROVED** [review]
LGTM

**Review by smangukiya — APPROVED** [review]
LGTM

**Pull request merged**